# Change the message displayed by Iridium's trace infobar to reflect prevent-trace-url-requests.patch

--- a/iridium/trkbar.cpp
+++ b/iridium/trkbar.cpp
@@ -29,7 +29,7 @@ TrkBar::TrkBar(const GURL &url) :
 
 base::string16 TrkBar::GetMessageText(void) const
 {
-	return base::ASCIIToUTF16("Loading traced URL: " + m_url.spec());
+	return base::ASCIIToUTF16("Blocked attempted request to: " + m_url.spec());
 }
 
 int TrkBar::GetButtons(void) const
